viewer.py

```diff
@@ -513,17 +530,7 @@ def main():
 
     # We don't want to show splash-page if there are active assets but all of them are not available
     view_image(LOAD_SCREEN)
-
-    logging.debug('Entering infinite loop.')
-    while True:
-        if loop_is_stopped:
-            sleep(0.1)
-            continue
-        if not db_conn:
-            load_settings()
-            db_conn = db.conn(settings['database'])
-
-        asset_loop(scheduler)
```

Comment:
@vpetersson, I had to move this block to a new function so that I can mock that function (`start_loop`) and so that the test will terminate. Otherwise, it'll never terminate/finish (infinite loop).